app/controllers/settings/recruitment/email_templates_controller.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class Settings::Recruitment::EmailTemplatesController < AuthorizedController
```

Comment:
What do you think about adding a test for either action?

Reply:
yeah, i have tests  for this controller but in another pr, will move part of them into this pr